dist/pathfora.js

```diff
@@ -3191,7 +3214,7 @@
      * @public
      * @description Current version
      */
-    this.version = '0.0.16';
+    this.version = '0.1.0';
```

Comment:
Ignore this, it should have been out with the last release. I forgot to run gulp after I bumped the version. :|